config/locales/en.yml

```diff
@@ -1322,6 +1322,14 @@ en:
   permission_view_wiki_edits: "View wiki history"
   permission_view_wiki_pages: "View wiki"
 
+  project:
+    destroy:
+      confirmation: "If you continue, the project %{identifier} and all related data will be permanently destroyed."
+      info: "Deleting the project is an irreversible action."
+      project_verification: "Enter the project's name %{name} to verify the deletion."
+      subprojects_confirmation: "Its subproject(s): %{value} will be also deleted."
```

Comment:
... will also be deleted? (word swapping)

Not sure, though. I am not to confident about english grammar...